corehq/apps/app_manager/tests/test_advanced_suite.py

```diff
@@ -196,6 +196,23 @@ def test_advanced_suite_load_case_from_fixture_with_custom_fixture(self):
         self.assertXmlPartialEqual(self.get_xml('load_case_from_custom_fixture_session'), suite, './entry[2]/session')
         self.assertXmlPartialEqual(self.get_xml('load_case_from_custom_fixture_instance'), suite, './entry[2]/instance')
 
+    def test_advanced_suite_load_case_from_fixture_with_report_fixture(self):
+        app = Application.wrap(self.get_json('suite-advanced'))
+        app.get_module(1).get_form(0).actions.load_update_cases.append(LoadUpdateAction(
+            case_tag="",
+            case_type="clinic",
+            load_case_from_fixture=LoadCaseFromFixture(
+                fixture_nodeset="instance('commcare:reports')/reports/report[@id='some-report-guid']/rows/row",
+                fixture_tag="selected_row",
+                fixture_variable="index",
+            )
+        ))
+        with patch('corehq.apps.app_manager.suite_xml.post_process.instances.toggles.MOBILE_UCR.enabled') as mock_toggle:
```

Comment:
I think you can use the [`flag_enabled`](https://github.com/dimagi/commcare-hq/blob/ed6846138b57ba8c370f01fe9cf72a613cf09ca3/corehq/util/test_utils.py#L119-L130) context manager